src/components/core/pivotTable/AggregationsSubMenu.tsx

```diff
@@ -58,10 +58,12 @@ export default class AggregationsSubMenu extends React.Component<IAggregationsSu
         rowAttributeHeaders: Execution.IAttributeHeader[],
         afmAttributeHeaderIndex: number
     ): string {
-        return afmAttributeHeaderIndex === 0
-            ? this.props.intl.formatMessage({ id: 'visualizations.menu.aggregations.all-rows' })
-            // subtotals are presented differently to the user and in AFM (menu displays the previous attribute name)
-            : this.getPreviousAttributeName(rowAttributeHeaders, afmAttributeHeaderIndex);
+        if (afmAttributeHeaderIndex === 0) {
+            return this.props.intl.formatMessage({ id: 'visualizations.menu.aggregations.all-rows' });
```

Comment:
I would use deconstruction for `intl` as it is used more than once, i.e.,
`const { intl } = this.props;

at the start of the method.

Reply:
fixed